web/src/components/Plugin/PluginDetail.tsx

```diff
@@ -243,6 +268,7 @@ const PluginDetail: React.FC<Props> = ({
         </Form>
         <Divider orientation="left">Data Editor</Divider>
         <PageHeader
+          className="test"
```

Comment:
Is the style sheet here added correctly?

Reply:
> Is the style sheet here added correctly?

good catch, this style sheet is useless, fixed, thanks a lot